parameters/experiments/m9/chinese_integrated_learner.params

```diff
@@ -0,0 +1,15 @@
+_includes:
+   - "../../root.params"
+   - "m9.params"
+
+experiment_group_dir: '%adam_experiment_root%m9/'
+hypothesis_log_dir: "%experiment_group_dir%hypotheses_recognizer/"
+log_hypothesis_every_n_steps: 25
+
+sort_learner_descriptions_by_length: True
+num_pretty_descriptions: 10
+
+experiment: 'm9-integrated-learner-given-object-recognizer'
+curriculum: 'm9-complete'
+learner: 'integrated-learner-recognizer'
+language_mode: CHINESE
```

Comment:
Can you remove this param from the PR? It needs to be added to the `readme.md` but we're going to request individuals add `language_mode: CHINESE` to their `root.params` to modify the language mode so we don't have to duplicate all our experiment param files.